vars/tdr.groovy

```diff
@@ -0,0 +1,11 @@
+def buildEndToEndTests(int delay, String buildJob, String[] params) {
+    build(job: buildJob, parameters: params, quietPeriod: delay, wait: false)
+}
+
+def setBuildStatus(String status, String desc) {
+    withCredentials([string(credentialsId: 'github-jenkins-api-key', variable: 'GITHUB_ACCESS_TOKEN')]) {
+        String sha = sh(script: 'git rev-parse HEAD', returnStdout: true).trim()
+        String url = "https://api.github.com/repos/nationalarchives/tdr-transfer-frontend/statuses/${sha}"
```

Comment:
This is hard-coded to the frontend URL. Perhaps the repo name should be a parameter?

Reply:
I've changed it to a parameter so it can be added to the URL now. I think to have it work with other repos we'll need to add permissions to the Jenkins GitHub user with the access key.

Reply:
OK, let me (or Sam or Tom) know when you need to update the other repos. 👍